### ISSUE TITLE
'colorcolumn' and 'hlsearch' Conflict

### ISSUE THREAD

**Issue by dylnmc** [arcticicestudio/nord-vim#118]
![meh](https://user-images.githubusercontent.com/7635158/39016219-bff165fa-43ed-11e8-9e83-05bb83881460.png)

There seems to be a conflict between the `hi ColorColumn` and `hi Search` groups when one has `:set colorcolumn=80` (for instance) and has searched for a term (and `:set hls` is enabled).

While I don't necessarily have time to look at this right now, I thought I would throw this up here since I noticed it.

Unfortunately, I am pretty sure there is no way to give precedence to highlight groups, but maybe there is something that can be done to fix this (doubtful). I didn't check previous bugs so sorry if this is a dup.

---

ps, it looks fine, obviously, when there is no `search` hl group on the colorcolumn .. just to be clear.

**Comment by dylnmc** [comment]
It looks like this is fixed with a later vim 8.0 patch. At work, `:version` yields:

```


VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Apr 23 2017 20:01:28)
MS-Windows 32-bit GUI version with OLE support
Included patches: 1-586
Compiled by mool@tororo
Huge version with GUI.  Features included (+) or not (-):
+acl                +digraphs           +langmap            -postscript         -termresponse
+arabic             +directx            +libcall            +printer            +textobjects
+autocmd            -dnd                +linebreak          +profile            +timers
+balloon_eval       -ebcdic             +lispindent         +python/dyn         +title
+browse             +emacs_tags         +listcmds           +python3/dyn        +toolbar
++builtin_terms     +eval               +localmap           +quickfix           +user_commands
+byte_offset        +ex_extra           -lua                +reltime            +vertsplit
+channel            +extra_search       +menu               +rightleft          +virtualedit
+cindent            +farsi              +mksession          +ruby/dyn           +visual
+clientserver       +file_in_path       +modify_fname       +scrollbind         +visualextra
+clipboard          +find_in_path       +mouse              +signs              +viminfo
+cmdline_compl      +float              +mouseshape         +smartindent        +vreplace
+cmdline_hist       +folding            +multi_byte_ime/dyn +startuptime        +wildignore
+cmdline_info       -footer             +multi_lang         +statusline         +wildmenu
+comments           +gettext/dyn        -mzscheme           -sun_workshop       +windows
+conceal            -hangul_input       +netbeans_intg      +syntax             +writebackup
+cryptv             +iconv/dyn          +num64              +tag_binary         -xfontset
+cscope             +insert_expand      +ole                +tag_old_static     -xim
+cursorbind         +job                +packages           -tag_any_white      +xpm_w32
+cursorshape        +jumplist           +path_extra         +tcl/dyn            -xterm_save
+dialog_con_gui     +keymap             +perl/dyn           -termguicolors      
+diff               +lambda             +persistent_undo    -tgetent            
   system vimrc file: "$VIM\vimrc"
     user vimrc file: "$HOME\_vimrc"
 2nd user vimrc file: "$HOME\vimfiles\vimrc"
 3rd user vimrc file: "$VIM\_vimrc"
      user exrc file: "$HOME\_exrc"
  2nd user exrc file: "$VIM\_exrc"
  system gvimrc file: "$VIM\gvimrc"
    user gvimrc file: "$HOME\_gvimrc"
2nd user gvimrc file: "$HOME\vimfiles\gvimrc"
3rd user gvimrc file: "$VIM\_gvimrc"
       defaults file: "$VIMRUNTIME\defaults.vim"
    system menu file: "$VIMRUNTIME\menu.vim"
Compilation: cl -c /W3 /nologo  -I. -Iproto -DHAVE_PATHDEF -DWIN32  -DFEAT_CSCOPE -DFEAT_NETBEANS_INTG -DFEAT_JOB_CHANNEL   -DFEAT_XPM_W32   -DWINVER=0x0501 -D_WIN32_WINNT=0x0501  /Fo.\ObjGXOLYHTRi386/ /Ox /GL -DNDEBUG  /Zl /MT -DFEAT_OLE -DFEAT_MBYTE_IME -DDYNAMIC_IME -DFEAT_GUI_W32 -DFEAT_DIRECTX -DDYNAMIC_DIRECTX -DDYNAMIC_ICONV -DDYNAMIC_GETTEXT -DFEAT_TCL -DDYNAMIC_TCL -DDYNAMIC_TCL_DLL=\"tcl86.dll\" -DDYNAMIC_TCL_VER=\"8.6\" -DFEAT_PYTHON -DDYNAMIC_PYTHON -DDYNAMIC_PYTHON_DLL=\"python27.dll\" -DFEAT_PYTHON3 -DDYNAMIC_PYTHON3 -DDYNAMIC_PYTHON3_DLL=\"python35.dll\" -DFEAT_PERL -DPERL_IMPLICIT_CONTEXT -DPERL_IMPLICIT_SYS -DDYNAMIC_PERL -DDYNAMIC_PERL_DLL=\"perl524.dll\" -DFEAT_RUBY -DDYNAMIC_RUBY -DDYNAMIC_RUBY_VER=22 -DDYNAMIC_RUBY_DLL=\"msvcrt-ruby220.dll\" -DFEAT_HUGE /Fd.\ObjGXOLYHTRi386/ /Zi
Linking: link /RELEASE /nologo /subsystem:windows /LTCG:STATUS oldnames.lib kernel32.lib advapi32.lib shell32.lib gdi32.lib  comdlg32.lib ole32.lib uuid.lib /machine:i386 gdi32.lib version.lib   winspool.lib comctl32.lib advapi32.lib shell32.lib  /machine:i386 /nodefaultlib libcmt.lib oleaut32.lib user32.lib     /nodefaultlib:python27.lib /nodefaultlib:python35.lib   "E:\tcl\lib\tclstub86.lib" WSock32.lib xpm\x86\lib\libXpm.lib /PDB:gvim.pdb -debug
```

**Comment by dylnmc** [comment]
sorry meant to close initially. I'm going to close now since I don't think there is anything we can do for older versions of vim
